eval_datasets/__init__.py

```diff
@@ -0,0 +1 @@
+__author__ = 'kudkudak'
```

Comment:
remove?

Reply:
let's add sourcing in README. If you adapted/fixed any details it may be nice to create a separate repo, might get you some stars.